Readme file (#23)
p1q/InternetShop: README.md

<p>You are free to use it as a template for your own online shop software. In current implementation you can add, remove and modify goods, users, roles and orders. The application provides registration and login procedures, passwords are saved into the database as hashes to increase security. You can make orders by adding goods into user's bucket.</p>

## :nut_and_bolt: Tech Stack
- Platform: Java EE 8
- Project management tool: Maven 4
- Database: MySQL 5
- ORM: Hibernate 5
- View level: JSP/JSTL
- Servlet container: Tomcat
- Logging: Log4j

## :rocket: Project Deployment
1. Install JDK latest version and set correct JAVA_HOME variable
2. Install Git (distributed version control system)
3. Install an IDE
4. Install and configure Apache Tomcat (I'm using version 8)
5. Install and configure MySQL database

> 6. Create a new schema 'internetshop' and import init_db.sql file into it

Reviewer: ```suggestion
6. Run init_db.sql file in your DB
```
